content/en/security/infrastructure_vulnerability/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Infrastructure Vulnerability Management
+kind: documentation
+is_beta: true
+further_reading:
+ - link: "/security/infrastructure_vulnerability/setup"
+   tag: "Documentation"
+   text: "Setting up Infrastructure Vulnerability Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/GwAWNEMSqyoao2fX6" >}}
+  Infrastructure Vulnerability Management is in private beta, but you can easily request access! Use this form to submit your request today.
+{{< /callout >}}
+
+Datadog [Infrastructure Vulnerability Management][1] uses the Datadog Agent to monitor your container images and identify known vulnerabilities for you to prioritize and remediate. The Agent scans running container images on hosts and creates a software bill of materials (SBOM). The SBOM is then scanned for vulnerabilities.
+
+## Review the health of your container images
+
+View a list of running container images on the [Container Images][2] page. Identify container images with vulnerabilities and view additional details about the vulnerability on the details page.
+
+{{< img src="security/infrastructure_vulnerability/container_images.png" alt="Container Images overview page" width="100%">}}
+
+## Identify the top container-based vulnerabilities
+
+Prioritize vulnerabilities for remediation based on severity on the [Vulnerabilities][3] page. View additional information about each vulnerability on the details page, including number of impacted images, the date on which the vulnerability was last detected, a description of the vulnerability, and recommended remediation steps.
+
+{{< img src="security/infrastructure_vulnerability/infra_vulnerability_explorer.png" alt="Infrastructure Vulnerability Management explorer page" width="100%">}}
+
```

Comment:
View additional information about each vulnerability by clicking on the vulnerability. This information includes information about affected container images, the date on which the vulnerability was last detected, a description of the vulnerability, and recommended remediation steps.

(Use attached screenshot)